libbeat/docs/howto/change-index-name.asciidoc

```diff
@@ -24,7 +24,7 @@ setup.template.pattern: "customname-*"
 -----
 
 ifndef::no_ilm[]
-WARNING: If <<ilm,index lifecycle management>> is enabled (which is typically the default), `setup.template.name` and `setup.template.pattern` are ignored.
+WARNING: If <<ilm,index lifecycle management>> is enabled (which is typically the default), `setup.template.name` and `setup.template.pattern` are ignored. Thus you need to set `setup.ilm.enabled: false` for it to work.
```

Comment:
You have an unclear antecedent in this sentence that makes it grammatically ambiguous. 

Maybe say something like: "Thus you must also set `setup.ilm.enabled: false` to disable index lifecycle management."